packages/profile-sync-controller/src/sdk/authentication-jwt-bearer/constants.ts

```diff
@@ -0,0 +1 @@
+export const SESSION_LIFETIME_MS = 45 * 60 * 1000; // 45 minutes in milliseconds
```

Comment:
Just to note, we will probably change this on SIWE flow (since we don't want users to sign in every 45 mins).

Lets keep a note of this, we can change this in a future version.

Reply:
Maybe we can make this dynamic - using the AccessToken `expiresIn` seconds to determine when we should wipe the session

Reply:
CC @dovydas55